docs/rest-api/concepts/scan-guidance.md

```diff
@@ -2,7 +2,7 @@
 SharePoint and OneDrive store millions of files.  It is critical to use the right calling patterns when trying to understand all files and changes at scale. Historically, there are many APIs to access files at an atomic level.  Many of these APIs are not efficient at a large scale but work well for a single user interaction. This guidance walks through how to monitor an Office 365 tenant or OneDrive so that your application integrates with Office 365 with maximum performance and efficiency. Applications that typically have this type of need are sync engines, backup providers, search indexers, classification engines, and data loss prevention providers. 
 
 ## Getting the right permissions
-To build trust with users it is important to use the right set of [permission scopes][] needed for an app to function.  Most scanning applications will want to operate with Application permissions, this indicates your application is running independently of any particular user.  To access files you should request either the **Files.Read.All** or **Files.ReadWrite.All** scope.  For access to SharePoint resources, **Sites.Read.All** or **Sites.ReadWrite.All** are appropriate.
+To build trust with users it is important to use the right set of [permission scopes][] needed for an app to function.  Most scanning applications will want to operate with Application permissions, this indicates your application is running independently of any particular user.  To access files you should request either the **Files.Read.All** or **Files.ReadWrite.All** scope.  For access to SharePoint resources, including the list of all site collections, **Sites.Read.All** or **Sites.ReadWrite.All** is appropriate.  In order to process permissions correctly you will also need to request **Sites.FullControl.All**.
```

Comment:
I wonder if we should say `minimal set` instead of `right set` to make it more clear that trust is built by NOT asking for permissions you don't need?